code/modules/trade/datums/trade_stations_presets/cybernetics.dm

```diff
@@ -0,0 +1,29 @@
+/datum/trade_station/cybermoebus
+	name_pool = list("FTS 'PentaOptium'" = "Free Trade Station 'PentaOptium'. \"Oh, you, i think i can sold you some cybernetics.\"")
```

Comment:
Many descriptions don't make much sense and don't follow the english language exactly, but i have been informed that these bing **soul** into the game so i'll let them pass. Also they are very funny.

Reply:
If you want to help with that, offer normal description, not a "This descriptions is bad, fix it"... :c

Reply:
I genuinely like these descriptions, and am not very good with English myself. When I had bad descriptions in my PR I got help from en docks to correct them. Though a few grammatically incorrect descriptions don't ruin a PR.